Makefile

```diff
@@ -440,6 +440,11 @@ build/toolchain/bin/golangci-lint$(EXE_EXTENSION):
 	mv $(TOOLCHAIN_DIR)/temp-golangci/golangci-lint$(EXE_EXTENSION) $(TOOLCHAIN_BIN)/golangci-lint$(EXE_EXTENSION)
 	rm -rf $(TOOLCHAIN_DIR)/temp-golangci/
 
+build/toolchain/python/:
+	# mkdir -p $(TOOLCHAIN_DIR)/python/
+	if ! [ -d "$(TOOLCHAIN_DIR)/python" ]; then virtualenv --python=python3 $(TOOLCHAIN_DIR)/python/; fi
```

Comment:
There's lines missing from the old code that need to be re-added back.

Reply:
I use the pip inside the virtualenv so that we could install the python module directly without activating the virtualenv itself. It seems like it is doing the same thing as installing after activating the virtualenv.